charts/fluid/fluid/templates/webhook/webhookconfiguration.yaml

```diff
@@ -46,4 +46,25 @@ webhooks:
     objectSelector:
       matchLabels:
         fuse.serverful.fluid.io/inject: "true"
+  - name: inplaceupdate.worker.fluid.io
+    rules:
+      - apiGroups: [ "" ]
+        apiVersions: [ "v1" ]
+        operations: [ "CREATE" ]
+        resources: [ "pods" ]
+    clientConfig:
+      service:
+        namespace: {{ include "fluid.namespace" . }}
+        name: fluid-pod-admission-webhook
+        path: "/mutate-fluid-io-v1alpha1-schedulepod"
+        port: 9443
+      caBundle: Cg==
+    timeoutSeconds: {{ .Values.webhook.timeoutSeconds }}
+    failurePolicy: Fail
+    reinvocationPolicy: {{ .Values.webhook.reinvocationPolicy }}
+    sideEffects: None
+    admissionReviewVersions: [ "v1","v1beta1" ]
+    objectSelector:
+      matchLabels:
+        fluid.io/runtime-pod-type: "worker"
```

Comment:
I think only Worker Pods requiring in-place updates trigger the mutating webhook, avoiding unnecessary webhook invocations for Pods that do not need in-place upgrades.

```golang
// When creating Worker Pods:
if runtime.Spec.UpdateStrategy == datav1alpha1.InPlaceUpdateStrategy || 
   runtime.Spec.UpdateStrategy == datav1alpha1.InPlaceIfPossibleUpdateStrategy {
    pod.Annotations["fluid.io/require-inplace-upgrade"] = "true"
}
```